### PR TITLE
🎨  sbpack fixes

### PR DESCRIPTION
**Pull request by dmiller15**
## Description

Updates the workflow so it can be pushed using sbpack.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] WGS Test: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/d8ea7ef8-b8c9-4694-91ea-4e4d891793de
- [x] WXS Test: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/d28424ac-ab36-453f-a730-8b1bb8a9d6d8
- [x] Tool Passes sbpack and cwltool validation
- [x] WXS Test for new flatten changes: https://cavatica.sbgenomics.com/u/kfdrc-harmonization/kf-reference-pipeline/tasks/c822fa30-34dc-4570-87d1-15f57f5d2644

**Test Configuration**:
* Environment:
* Test files:

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have committed any related changes to the PR

### CODE REVIEW

**Review by migbro — APPROVED** [review]
This is basic and small enough where I am ok not seeing a test run.

File: tools/sambamba_merge_anylist.cwl [hunk 1]
```diff
@@ -45,7 +45,7 @@ arguments:
         }
       }
 inputs:
-  bams: { type: 'Any[]', doc: "Input will be a combination of File[] and deeply nested File[]. Any[] is flexible enough to accomodate the different types" }
+  bams: { type: 'File[]', doc: "Input will be a combination of File[] and deeply nested File[]. Any[] is flexible enough to accomodate the different types" }
```
Comment (sickler-alex) [inline tools/sambamba_merge_anylist.cwl:48]: You can remove the description about "Any[]" since it's not used anymore.

**Review by sickler-alex — APPROVED** [review]

**Comment by dmiller15** [comment]
Figured out the issue cwltool was highlighting and fixed it.

File: tools/sambamba_merge_anylist.cwl (hunk 1 above)
Comment (dmiller15) [reply tools/sambamba_merge_anylist.cwl]: Done

**Pull request merged**